dts/bindings/ethernet/microchip,lan8720a.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023 Basalte bv
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip LAN8720A PHY
+
+compatible: "microchip,lan8720a"
+
+include: phy.yaml
+
+on-bus: mdio
+
+properties:
+  reg:
+    required: true
+
+  reset-gpios:
+    type: phandle-array
+    description: Reset pin.
+
+      The reset pin of LAN8720A is active low.
+      If connected directly the MCU pin should be configured
+      as active low.
+
+  irq-gpios:
+    type: phandle-array
+    required: true
```

Comment:
Is it really true? I have a board where the INT signal is floating.

![image](https://github.com/zephyrproject-rtos/zephyr/assets/8523351/ff56ab7e-3e40-4f14-aa70-680e0ad5c29d)

Reply:
It probably isn't, but my current implementation depends on it, as we have this connected.

Reply:
So this must be changed into optional since it will be non-usable for boards like mine.

Reply:
It will require additional charges, yes